gmail/style.css

```diff
@@ -375,11 +375,11 @@ html.simpl .aeF {
 /* Max width inbox and conversation and add top padding */
 html.simpl .AO .aeF > .nH {
 	margin: 40px auto 0 auto;
-	max-width: 960px; 
+	max-width: 960px;
 }
 @media only screen and (min-width: 1450px) {
```

Comment:
Why this is `min-width: 1450px`? I have a laptop with a resolution of `1366x768`. I have tested this branch and I wonder if we can change the `min-width` to `1360px`, it works very well on my machine.